llmfoundry/__init__.py

```diff
@@ -9,6 +9,23 @@
 
 import logging
 
+try:
+    from flash_attn import flash_attn_func
+    del flash_attn_func
+except ImportError as e:
+    if 'undefined symbol' in str(e):
+        raise ImportError(
+            'The flash_attn package is not installed correctly. Usually this means that your runtime version.'
+            +
+            ' of PyTorch is different from the version that flash_attn was installed with, which can occur when your'
+            +
+            ' workflow has resulted in PyTorch being reinstalled. This probably happened because you are using an old docker image'
```

Comment:
```suggestion
            ' workflow has resulted in PyTorch being reinstalled. This probably happened because you are using an old Docker image'
```